mobile/src/Student/containers/TeamIcons/img/index.js

```diff
@@ -0,0 +1,158 @@
+import { useState } from 'react'
+import { FlatList, Image, SafeAreaView, StyleSheet, Text, View } from 'react-native'
+import { scale, verticalScale } from 'react-native-size-matters'
+import Pressable from 'react-native/Libraries/Components/Pressable/Pressable'
+import PurpleBackground from '../../../components/PurpleBackground'
+import RoundButton from '../../../components/RoundButton'
+import { colors, fontFamilies, fonts } from '../../../utils/theme'
+import TeamIcons from '../../containers/TeamIcons'
+
+const SelectTeam = ({ navigation, team, saveTeamAvatar }) => {
```

Comment:
Is this a new component? I couldn't find the place that it's used in the PR. Is this used in this PR or intended for a future PR?

Reply:
This is because the location of the images was changed with new exports